app/views/blog/index.html.erb

```diff
@@ -1,3 +1,10 @@
 <h2> My blog Tracker</h2>
+<br>
 
-<%# @blogs.each do |blog| %>
+<ul>
+
+<% @blogs.each do |blog| %> 
+<li> <%= link_to blog.title, blog_path(blog.id) %> </li>
+<% end %> 
```

Comment:
Keep an eye on your indentation here.  Any tag nested within another tag should be tabbed over.

Reply:
I just saw that thank you! Will be fix on the next push